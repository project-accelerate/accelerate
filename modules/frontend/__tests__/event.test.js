/* eslint-env jest */

import React from "react";
import faker from "faker/locale/en_GB";
import renderer from "react-test-renderer";
import EventPage from "../pages/event";

describe("/event", () => {
  it("should match snapshot", () => {
    const tree = renderer.create(
      <EventPage
        event={{
          title: faker.lorem.words(3),
          address: faker.address.streetAddress(),
          organiser: faker.lorem.words(3),
          startDate: "2018-01-23T20:00:00.000Z",
          endDate: "2018-01-23T22:00:00.000Z",
          longDescription: faker.lorem.paragraphs(5),
          postcode: faker.address.zipCode()
        }}
      />
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
