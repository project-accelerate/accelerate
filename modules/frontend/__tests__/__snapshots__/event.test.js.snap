// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/event should match snapshot 1`] = `
<div
  className="MuiPaper-root-4 MuiPaper-shadow2-8 MuiPaper-rounded-5"
>
  <div
    className="MuiCardContent-root-31"
  >
    <h1
      className="MuiTypography-root-32 MuiTypography-headline-37 EventPage-heading-2"
    >
      optio quo quis
    </h1>
    <h2
      className="MuiTypography-root-32 MuiTypography-body1-41 EventPage-heading-2"
    >
      Organised by 
      <strong>
        dolor voluptatibus velit
      </strong>
    </h2>
    <h3
      className="MuiTypography-root-32 MuiTypography-body1-41 EventPage-heading-2"
    >
      Tue 23rd Jan, 8:00pm - 10:00pm
    </h3>
    <p
      className="MuiTypography-root-32 MuiTypography-body1-41 EventPage-heading-2"
    >
      <span>
        2657 Jones Roads
        <br />
      </span>
      RX52 3JY
    </p>
    <section
      className="EventPage-section-3"
    >
      <p
        className="MuiTypography-root-32 MuiTypography-body2-40 EventPage-paragraph-1"
      >
        Deleniti eos reprehenderit in nisi et qui. Odio et inventore eligendi in. Id harum sit odio quia vitae provident. Provident molestiae harum ullam pariatur quos est quod. Ea quisquam esse quia expedita commodi autem.
      </p>
      <p
        className="MuiTypography-root-32 MuiTypography-body2-40 EventPage-paragraph-1"
      >
         
Exercitationem doloribus harum voluptas. Omnis et accusantium. Quia quos sequi molestiae beatae enim.
      </p>
      <p
        className="MuiTypography-root-32 MuiTypography-body2-40 EventPage-paragraph-1"
      >
         
Placeat saepe eligendi rerum excepturi et. Fuga similique fugit corporis voluptatem dolores. Nobis et sint magnam aut atque. Maiores incidunt occaecati iusto laborum facere. Deleniti dolorem quibusdam. Veniam perspiciatis et nisi aut corporis laboriosam.
      </p>
      <p
        className="MuiTypography-root-32 MuiTypography-body2-40 EventPage-paragraph-1"
      >
         
Soluta accusamus officiis ut excepturi blanditiis libero ut. Quo fuga enim tempore non voluptas quia. Ipsam voluptas necessitatibus. Dolor adipisci dolores sunt.
      </p>
      <p
        className="MuiTypography-root-32 MuiTypography-body2-40 EventPage-paragraph-1"
      >
         
Occaecati rerum nesciunt. Sint est asperiores sit voluptatum mollitia enim iste. Nesciunt minima sequi voluptas optio aut voluptatem.
      </p>
    </section>
  </div>
</div>
`;
